index.html

```diff
@@ -206,15 +206,15 @@ <h2 class="about-title">Skills</h2>
       <h2 class="contact-h2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
       <form id="form"  action="https://formspree.io/f/mbjelbzj" method="POST">
         <div class="input-control">
-            <input id="fullname" name="fullname" type="text" placeholder="Full name">
+            <input id="fullname" name="fullname" type="text" placeholder="Full name" maxlength="30">
             <div class="error"></div>
         </div>
         <div class="input-control">
             <input id="email" name="email" type="text" placeholder="Email address">
             <div class="error"></div>
         </div>
         <div class="input-control">
-          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] While making changes as requested by the previous code reviewer, your project design does not match the Figma design. Please make sure that you adjust the style of your project to look exactly like the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=ZV2gXsCnfUjJSp7y-0)
### Your design
![Screenshot from 2023-01-23 22-20-49](https://user-images.githubusercontent.com/92303891/214164146-da122ef2-6249-445b-a710-025c468604e6.png)

### Figma design
![Screenshot from 2023-01-23 22-22-49](https://user-images.githubusercontent.com/92303891/214164188-2fed8ba5-47cf-440f-99f0-d250298f0cea.png)

Reply:
@Owusu-Desmond  This is a design for desktop view , This review was for mobile